package.json

```diff
@@ -78,8 +78,8 @@
     "concurrently": "^8.2.0",
     "eslint": "^8.4.1",
     "eslint-config-prettier": "^9.0.0",
+    "eslint-plugin-import": "^2.29.1",
     "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-simple-import-sort": "^10.0.0",
```

Comment:
dev: this is largely a subset (in terms of functionality) of `eslint-plugin-import`, thus it's not necessary anymore